leaf-analysis/app.js

```diff
@@ -113,5 +113,3 @@ function passIntoJar(res) {
 }
 

```

Comment:
Can you please remove this file from your commit, thanks

Reply:
Also deleting this line makes it impossible to actually run (and therefore test) the web page.